app_dart/test/request_handlers/get_status_test.dart

```diff
@@ -55,11 +55,13 @@ void main() {
       final Commit commit1 = Commit(
           key: config.db.emptyKey.append(Commit, id: 'flutter/flutter/ea28a9c34dc701de891eaf74503ca4717019f829'),
           timestamp: 3,
-          branch: 'master');
+          branch: 'master',
```

Comment:
nit: prefer `config.defaultBranch` (for inclusivity purposes and not to hard code the master branch since we'll want to move to main)